src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -22,29 +22,6 @@
 __all__ = ["UVBeam"]
 
 
-def _convert_feeds_to_pols(feed_array, calc_cross_pols, x_orientation=None):
```

Comment:
I get this was a "private" function before, but with how python works, should we leave this function and issues a big ol' warning to the user about the deprecation instead?

Reply:
That's fair. I can add that back in.

Reply:
done.